networkdb/networkdb.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/Sirupsen/logrus"
 	"github.com/armon/go-radix"
 	"github.com/docker/go-events"
+	"github.com/docker/libnetwork/types"
```

Comment:
`networkdb` should not import some libnetwork internal package. We should keep `networkdb` independent of rest of libnetwork. Can we define `PeerInfo` in `networkdb` itself?